src/aosfxplayer.cpp

```diff
@@ -37,11 +37,16 @@ void AOSfxPlayer::play(QString p_sfx, QString p_char, QString shout,
 
   QString misc_path = "";
   QString char_path = "";
+  QString theme_path = "";
   QString sound_path = ao_app->get_sfx_suffix(ao_app->get_sounds_path(p_sfx));
 
-  if (shout != "")
+  if (shout != "") {
     misc_path = ao_app->get_sfx_suffix(ao_app->get_base_path() + "misc/" +
                                        shout + "/" + p_sfx);
+    theme_path = ao_app->get_sfx_suffix(ao_app->get_theme_path(p_sfx));
+    if (!file_exists(theme_path))
+      theme_path = ao_app->get_sfx_suffix(ao_app->get_default_theme_path(p_sfx));
+  }
```

Comment:
So did it never check the theme in the first place?

Reply:
Seems like it.